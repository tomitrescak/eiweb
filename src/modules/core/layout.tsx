import * as React from 'react';

export const Layout = ({ children }: any) => {
return children;
}

Layout.displayName = 'Layout';
