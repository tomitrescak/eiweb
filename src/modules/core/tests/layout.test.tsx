import * as React from 'react';

import { mount } from 'enzyme';

import { Layout } from '../layout';


const Component = () => <div>Rendered</div>

describe('Layout @view', () => {
  it('renders child component', () => {
    expect(true).toBe(true);
    const m = mount(<Layout><Component /><div>ee</div></Layout>);
    
    expect(m.html()).toMatchSnapshot(); 
  });
});
