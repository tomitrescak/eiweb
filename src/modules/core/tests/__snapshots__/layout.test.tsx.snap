// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout @view renders child component 1`] = `"<div><div>Rendered</div><div>ee</div></div>"`;
